app/src/marketplace/i18n/en.po

```diff
@@ -250,7 +250,7 @@ msgid "editProductPage.sidebar.activeSubscribers"
 msgstr "Active subscribers"
 
 msgid "editProductPage.sidebar.mostRecentSubscription"
-msgstr "Most recent subscription"
+msgstr "Most recent purchase"
```

Comment:
This is how it's supposed to be in the design, even though other instances were renamed to "subscription". Using "Most recent subscription" pushes it to the next line 🙄

Reply:
Is the line break the only concern here? Sounds like we're going with subs these days, not purchases, and this kinda takes us a step back.

Reply:
@mattatgit Is this intentional or change to "Most recent subscription"?

![Screenshot 2020-10-26 at 12 03 21](https://user-images.githubusercontent.com/1064982/97159273-47f00900-1783-11eb-8d3a-8ed0132780c5.png)